vyxal/elements.py

```diff
@@ -3910,14 +3966,14 @@ def vy_repr(lhs, ctx):
         # actually make the repr kinda make sense
     }.get(
         ts,
-        lambda: "⟨ "
-        + " | ".join(
+        lambda: ("⟨ " if ctx.vyxal_lists else "[")
```

Comment:
vy_str and vy_repr seem to repeat code for lists. Maybe extract this bit into a helper or something?

Reply:
It'd be too specialised for a helper - also it's purely for flag purposes.